lib/pages/auto_homeserver_picker/auto_homeserver_picker_view.dart

```diff
@@ -17,8 +17,8 @@ class AutoHomeserverPickerView extends StatelessWidget {
     return Scaffold(
       body: Container(
         width: MediaQuery.of(context).size.width,
-        decoration: BoxDecoration(
-          gradient: LinagoraSysColors.material().linearGradientStartingPage,
+        decoration: const BoxDecoration(
+          color: Colors.white,
```

Comment:
I think this should be in AutoHomeserverPickerViewStyle

Reply:
done